src/extractors/generic/index.test.js

```diff
@@ -8,7 +8,10 @@ const fs = require('fs');
 describe('GenericExtractor', () => {
   describe('extract(opts)', () => {
     it('extracts this old LA Times article', () => {
-      const html = fs.readFileSync('./fixtures/latimes.html', 'utf-8');
+      const html = fs.readFileSync(
+        './fixtures/www.latimes.com--old.html',
+        'utf-8'
+      );
```

Comment:
tests passed, cat `fixtures/www.latimes.com--old.html` also works 👍🏽

should we add a test here for the regular extractor for the la-times to show that the regular url will fail since it has a custom extractor or is that basically inferred bc of the `custom/www.latimes.com/index.test.js` test?

Reply:
With this one, I decided to keep the old fixture as the basis for the generic extractor tests to show that the generic extractor functionality didn't change at all. I do think that the tests for the custom extractor sufficiently cover the new HTML in the updated fixture.

Thanks for being so thorough -- it is much, much appreciated!